ocaml/xapi-cli-server/record_util.ml

```diff
@@ -1241,3 +1241,16 @@ let vm_placement_policy_of_string a =
       `anti_affinity
   | s ->
       raise (Record_failure ("Invalid VM placement policy, got " ^ s))
+
+let repo_origin_to_string = function `remote -> "remote" | `bundle -> "bundle"
+
+let repo_origin_of_string = function
```

Comment:
I've changed this module on `master`; rebase and notice that raising an error has become simpler.